src/index.ts

```diff
@@ -64,6 +63,7 @@ export async function start(): Promise<void> {
 
   process.on('uncaughtException', (error: Error) => {
     // I just received an error that was never handled, time to handle it and then decide whether a restart is needed
+    promErrorsCounter.inc(1);
```

Comment:
I'm not sure this one is very useful :)

Since you're exiting right after, it will never be scrapped (prometheus metrics are not persisted to disk)